portality/bll/services/application.py

```diff
@@ -251,6 +251,22 @@ def accept_application(self, application, account, manual_update=True, provenanc
 
         return j
 
+    def reject_update_request_of_journal(self, journal_id, account):
```

Comment:
Can we have a unit test or two for this function please

Reply:
Already implemented.